packages/openapi3/src/lib.ts

```diff
@@ -275,13 +275,27 @@ export const libDef = {
         default: paramMessage`XML \`@unwrapped\` can only used on array properties or primitive ones in the OpenAPI 3 emitter, Property '${"name"}' will be ignored.`,
       },
     },
+    "duplicate-tag": {
+      severity: "error",
+      messages: {
+        default: paramMessage`"Duplicate tag '${"tagName"}' found. Tag names must be unique in OpenAPI3."`,
```

Comment:
```suggestion
        default: paramMessage`"Metadata for tag '${"tagName"}' was specified twice."`,
```

I woudl maybe just even remove this and use `validateDecoratorUniqueOnNode` instead.